.github/workflows/cron-weekly.yaml

```diff
@@ -21,6 +21,7 @@ jobs:
           use-verbose-mode: "yes"
           config-file: ".github/workflows/mlc_config.json"
           folder-path: "docs/"
+          file-path: "./README.md"
```

Comment:
Isn't there a conflict if we have `folder-path` and `file-path` defined together?

Reply:
No, this action allows us to specify extra files outside of `folder-path`
see: https://github.com/gaurav-nelson/github-action-markdown-link-check/tree/v1/#check-multiple-directories-and-files